src/scancode_config.py

```diff
@@ -75,8 +75,11 @@ def _create_dir(location):
 
 try:
     from pkg_resources import get_distribution, DistributionNotFound
-    __version__ = get_distribution('scancode-toolkit').version
-except (DistributionNotFound, ImportError):
+    try:
+        __version__ = get_distribution('scancode-toolkit').version
+    except DistributionNotFound:
+        __version__ = '21.3.31'
```

Comment:
I wonder if we could may be define this default before the try block and only once instead?

Reply:
Sure we could - will spin up a v2 later

Reply:
Is this better now?